src/coreclr/vm/interpreter.cpp

```diff
@@ -12022,7 +12022,7 @@ const char* Interpreter::getMethodName(CEEInfo* info, CORINFO_METHOD_HANDLE hnd,
         return pMD->GetName();
     }
 
-    return info->getMethodNameFromMetadata(hnd, className, namespaceName, enclosingClassName);
+    return info->getMethodNameFromMetadata(hnd, className, namespaceName, enclosingClassName, 0);
```

Comment:
Should we pass in 1 instead of 0? Zero seems to indicate that no enclosingClassName should be extracted.

Reply:
That doesn't work, it will lead to an AV, it looks like that number is used as a buffer size somewhere.

Reply:
Ah, the enclosingClassName can be NULL, in that case passing in non-zero is wrong. So I'd pass 0 or 1 depending on the enclosingClassName  being NULL or not.

Reply:
All the callers are passing `NULL` for `enclosingClassName`

Reply:
I have discovered that too. But I'd rather make this able to handle the case when it is not NULL or remove the argument from the `Interpreter::getMethodName` completely to avoid surprises in the future when someone would want to start passing non-null and it would not work as expected.